packages/notification-service/src/polling.ts

```diff
@@ -3,11 +3,13 @@ import { handleTransferNotifications } from './blockscout/transfers'
 import { EXCHANGE_POLLING_INTERVAL, POLLING_INTERVAL } from './config'
 import { handleExchangeQuery } from './exchange/exchangeQuery'
 import { handlePaymentRequests } from './handlers'
+import { handleInvites } from './invites/invites'
 
 export const notificationPolling = AsyncPolling(async (end) => {
   try {
     await handleTransferNotifications()
     await handlePaymentRequests()
+    await handleInvites()
```

Comment:
Should we do these in parallel?

Reply:
Yeah, added a different poller for invites with a slightly longer interval.